src/main/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperService.java

```diff
@@ -143,6 +146,7 @@ void setEnrollmentOptionsFactory(EnrollmentOptionsFactory enrollmentOptionsFacto
         try{
             return assignmentReader.getSingleAssignment(singleAssignmentOptions);
         } catch(Exception e){
+            LOG.info("Assignment not found. " + e);
```

Comment:
which assignment? who was the user? what course? These are the types of things we need to know to make logging messages worth anything.